packages/@aws-cdk/custom-resources/lib/aws-custom-resource/aws-custom-resource.ts

```diff
@@ -2,8 +2,12 @@ import { CustomResource, CustomResourceProvider } from '@aws-cdk/aws-cloudformat
 import iam = require('@aws-cdk/aws-iam');
 import lambda = require('@aws-cdk/aws-lambda');
 import cdk = require('@aws-cdk/core');
+import fs = require('fs');
 import path = require('path');
-import metadata = require('./sdk-api-metadata.json');
+
+// don't use "require" since the typescript compiler emits errors since this
+// file is not listed in tsconfig.json.
```

Comment:
Why not add it to `tsconfig.json`?

Reply:
Because it is generated by jsii

Reply:
Maybe we should add a feature request to `jsii` to support adding files to the `includes` clause... We already support injecting `exclude` clauses......

Reply:
https://github.com/aws/jsii/issues/965